pom.xml

```diff
@@ -78,6 +79,13 @@
               </dependency-resolution>
             </configuration>
           </plugin>
+          <plugin>
+            <groupId>org.eclipse.tycho</groupId>
+            <artifactId>tycho-p2-plugin</artifactId>
+            <configuration>
+              <baselineReplace>${p2-baseline-replace-mode}</baselineReplace>
+            </configuration>
+          </plugin>
```

Comment:
With https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/pull/1736, this extra configuration would not be necessary.